src/mutation.ts

```diff
@@ -116,6 +116,11 @@ export class Mutation {
 
       if (Number.isSafeInteger(num)) {
```

Comment:
I think this is a nonissue here, but wanted to raise it anyway just in case. We have had issues in other libraries with the single Number type in node.js. Reading this code, a double of `3.00...` seems like it would be interpreted as if it was the long 3. Seeing as this method returns the node numbers this is likely not an issue. It may be though if we have a to buffer method?